apps/condo/domains/analytics/components/charts/resident/ResidentByPropertyChart.tsx

```diff
@@ -1,36 +1,116 @@
 import { Row, Col } from 'antd'
-import React, { useMemo } from 'react'
+import get from 'lodash/get'
+import React, { useMemo, useState, useEffect } from 'react'
 
+import { useLazyQuery } from '@open-condo/next/apollo'
 import { useIntl } from '@open-condo/next/intl'
-import { Typography } from '@open-condo/ui'
+import { Typography, Tooltip } from '@open-condo/ui'
 
-import { CustomChartView } from '@condo/domains/analytics/components/CustomChartView'
+import { CustomChartView, CHART_CONTAINER_HEIGHT } from '@condo/domains/analytics/components/CustomChartView'
+import { CustomListView } from '@condo/domains/analytics/components/CustomListView'
+import { GET_OVERVIEW_DASHBOARD_MUTATION } from '@condo/domains/analytics/gql'
+import { usePropertyFilter, useDateRangeFilter } from '@condo/domains/analytics/hooks/useDashboardFilters'
+import { useDetailChartView } from '@condo/domains/analytics/hooks/useDetailChartView'
+import { getClientSideSenderInfo } from '@condo/domains/common/utils/userid.utils'
 
 import { ResidentByPropertyDataMapper } from './dataMappers'
 
-import type { IResidentChartCard } from './dataMappers'
+import type { ResidentChartCardType } from './dataMappers'
 
-const ResidentByPropertyChart: IResidentChartCard = ({ data }) => {
+const ResidentByPropertyChart: ResidentChartCardType = ({ data, organizationId }) => {
     const intl = useIntl()
     const ChartTitle = intl.formatMessage({ id: 'pages.reports.residentsTitle' })
     const ResidentTitle = intl.formatMessage({ id: 'global.section.contacts' })
+    const AddressTitle = intl.formatMessage({ id: 'field.Address' })
+    const CountTitle = intl.formatMessage({ id: 'global.count' })
+    const ResidentColumnTitle = intl.formatMessage({ id: 'pages.condo.analytics.residentsPercent.title' })
+    const ResidentColumnDescription = intl.formatMessage({ id: 'pages.condo.analytics.residentsPercent.description' })
+
+    const [localData, setLocalData] = useState([])
+
+    const { values: propertyIds, SearchInput } = usePropertyFilter({ organizationId })
+    const { dateRange } = useDateRangeFilter()
+    const { open, isOpen, PopupChartView } = useDetailChartView({ title: ChartTitle })
+    const [loadResidentData, { loading }] = useLazyQuery(GET_OVERVIEW_DASHBOARD_MUTATION, {
+        onCompleted: (response) => {
+            setLocalData(get(response, 'result.overview.resident.residents', []))
+        },
+        onError: error => {console.log(error)},
```

Comment:
Are we will get empty data if there is an error? Is there an ability to display "oops, something wrong"?

Reply:
Yes, I'll think about it